ktor-server/ktor-server-jetty-jakarta/jvm/test/io/ktor/tests/server/jetty/jakarta/JettyServletApplicationEngine.kt

```diff
@@ -51,29 +54,28 @@ internal class JettyServletApplicationEngine(
     async: Boolean
 ) : JettyApplicationEngineBase(environment, monitor, developmentMode, configuration, applicationProvider) {
     init {
-        server.handler = ServletContextHandler().apply {
-            classLoader = environment.classLoader
+        server.handler = ServletContextHandler().apply<ServletContextHandler> {
```

Comment:
Minor. Explicit generics and `this.` seem redundant here and in other places in this file